jetstream/cli.py

```diff
@@ -239,52 +277,65 @@ def convert(self, value, param, ctx):
 )
 @experiment_slug_option
 @secret_config_file_option
-def run(project_id, dataset_id, date, experiment_slug, config_file):
-    """
-    Runs analysis on active experiments for the provided date.
-
-    This command is invoked by Airflow. All errors are written to the console and
-    BigQuery. Runs will always return a success code, even if exceptions were
-    thrown during some experiment analyses. This ensures that the Airflow task will
-    not retry the task and run all of the analyses again.
-    """
-    AnalysisExecutor(
+@argo_option
+@zone_option
+@cluster_id_option
+@monitor_status_option
+def run(
```

Comment:
If we end up making --argo and the config file option mutually exclusive, I would advocate for a separate argo_run() command that makes it impossible to specify them both. That also clarifies which of these options are only meaningful in conjunction with --argo.

Reply:
I added a `argo-run` command